litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/__tests__/WorkflowSettings.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { screen, fireEvent } from '@testing-library/react';
+import { render } from '../../../../testHelpers/test-util';
+import WorkflowSettings from '../index';
+import '@testing-library/jest-dom';
+
+describe('WorkflowSettings', () => {
+  it('Renders', () => {
+    render(<WorkflowSettings />);
+
+    const inputs = screen.getAllByRole('textbox');
+
+    inputs.forEach((input) => {
+      expect(input).toHaveProperty('disabled', false);
+      fireEvent.change(input, { target: { value: 'random text' } });
```

Comment:
just checking this will not be of much value because these tests are already done in litmus-ui...fo rather than checking the input field ...we should think of test cases that check the components working.

Reply:
Will try to incorporate that!